Exercise3/Models/ManagerItem.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Runtime.Serialization;
+
+namespace Exercise3.Models
+{
+    public class ManagerItem : ISerializable
```

Comment:
Why you need to implement `ISerializable`

Reply:
My bad, I forgot to delete it.
Done!